index.js

```diff
@@ -3,53 +3,183 @@ const state = {
     {
       id: "001-beetroot",
       name: "beetroot",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "002-carrot",
       name: "carrot",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "003-apple",
       name: "apple",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "004-apricot",
       name: "apricot",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "005-avocado",
       name: "avocado",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "006-bananas",
       name: "bananas",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "007-bell-pepper",
       name: "bell pepper",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "008-berry",
       name: "berry",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "009-blueberry",
       name: "blueberry",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "010-eggplant",
       name: "eggplant",
-      price: 0.35
-    }
+      price: 0.35,
+    },
   ],
-  cart: []
-};
\ No newline at end of file
+  cart: [],
+};
+
+// for all the items in const state, render it as a li to replicates store-item.html in header (#id=store)
+// all the items should contain appropriate image and a button "add to cart" inside li
+const itemList = document.querySelector(".store--item-list");
+console.log(itemList);
+function calculateTotal() {
+  // this function should calculate total of the cart
+  let cartTotal = 0;
+  // calculate total
+
+  // display total
+  return cartTotal;
+}
+
+createItemList();
```

Comment:
Very nice work Abul! I know this was quite difficult but you got there in the end. Keep it up!